pages/aer.vue

```diff
@@ -85,12 +85,12 @@ print(sim_result.get_counts(qc))</SyntaxHighlight>
 
 <script lang="ts">
 import { Component } from 'vue-property-decorator'
-import QiskitElementPage from '~/components/qiskit/QiskitElementPage.vue'
-import LegacyPresentation from '~/components/headers/LegacyPresentation.vue'
-import LegacySection from '~/components/sections/LegacySection.vue'
-import SoftwareStack from '~/components/qiskit/SoftwareStack.vue'
-import SyntaxHighlight from '~/components/qiskit/SyntaxHighlight.vue'
-import Cta from '~/components/ctas/Cta.vue'
+import QiskitElementPage from '~/components/logic/QiskitElementPage.vue'
+import LegacyPresentation from '~/components/elements/LegacyPresentation.vue'
+import LegacySection from '~/components/ui/sections/LegacySection.vue'
+import SoftwareStack from '~/components/ui/SoftwareStack.vue'
```

Comment:
Non-blocking: the "`SoftwareSackp`" is very tied to the elements and I'd put it there.